docs/reference/migration/migrate_7_0/search.asciidoc

```diff
@@ -15,6 +15,9 @@
 *   The boundary specified using geohashes in the `geo_bounding_box` query
     now include entire geohash cell, instead of just geohash center.
 
+*   Attempts to generate multi-term phrase queries against non-indexed fields
+    will now throw an exception
```

Comment:
Only if you force an analyzer for all fields in the query ? This limits the scope of the breaking change so I think it's important to mention it.

Reply:
also maybe "non-indexed" should be "non-text" since eg. dates are conceptually indexed, just differently. Or alternatively "against fields that do not index positions"